### ISSUE TITLE
Form fields need semantic association to instructions when present

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2193]
Wherever form fields have instructional text, they must share an association using the `aria-describedby` attribute.

**Comment by avaaec** [comment]
Unable to find any occurrences of this issue, seems to be fixed.
